concent_api/core/validation.py

```diff
@@ -218,3 +225,81 @@ def validate_list_task_to_compute_ids(subtask_results_accepted_list):
     for task_to_compute in subtask_results_accepted_list:
         subtask_ids.append(task_to_compute.subtask_id + ':' + task_to_compute.task_id)
     return len(subtask_ids) == len(set(subtask_ids))
+
+
+def validate_file_transfer_token(file_transfer_token):
+    """
+    Function for check ConcentFileTransferToken each field:
+    """
+    # -SIGNATURE
+    if not isinstance(file_transfer_token.sig, bytes):
+        return ('Empty signature field in FileTransferToken message.', ErrorCode.MESSAGE_SIGNATURE_MISSING)
+    # -DEADLINE
+    if not isinstance(file_transfer_token.token_expiration_deadline, int):
+        return ('Wrong type of token_expiration_deadline field value.', ErrorCode.MESSAGE_TOKEN_EXPIRATION_DEADLINE_WRONG_TYPE)
+    current_time = int(timezone.now().timestamp())
```

Comment:
Please add an empty line after return.

Reply:
I wasn't able to do this that way, because there is `from core.validation import ...` in utils.helpers. And that cause import error.

Reply:
This is called `circular import` and has to be resolved here. You have to move the common code that uses those imports into separate module. So the `get_validated_client_public_key_from_client_message` which uses `from core.validation                import validate_public_key` has to be placed elsewhere.

Reply:
You still didn't replaced `int(timezone.now().timestamp())`.